doc/design/adding-pod-disruption-budgets.md

```diff
@@ -0,0 +1,45 @@
+# Adding Pod Disruption Budgets
+
+## Description
+
+OLM supports users including `PodDisruptionBudget` (PDB) objects in their bundle alongside their operator manifests. `PodDisruptionBudgets`
+are used to provide detailed information to the kube-scheduler about how many pods in a collection can be available or unavailable at given time. 
+For more info, see the docs at https://kubernetes.io/docs/tasks/run-application/configure-pdb/#protecting-an-application-with-a-poddisruptionbudget. 
+
+## Caveats
+
+PDBs are namespaced resources that only affect certain pods selected by the pod selector. However, 
+setting `maxUnavailable` to 0 or 0% (or `minAvailable` to 100%) on the PDB means zero voluntary evictions. 
+This can make a node impossible to drain and block important lifecycle actions like operator upgrades or even cluster upgrades. 
+
+Multiple PDBs can exist in one namespace- this can cause conflicts. Be sure to know of existing PDBs in the namespace in which
+your operator and operands will exist in the cluster. 
+
+PDBs for pods controlled by operators have additional restrictions around them. See https://kubernetes.io/docs/tasks/run-application/configure-pdb/#arbitrary-controllers-and-selectors
+for additional details - PDBs for operands managed by OLM-installed operators will fall into these restrictions. 
+
+## Technical Details
+
+PDB yaml manifests can be placed in the bundle alongside existing manifests in the `/manifests` directory. The PDB manifest will be present
+in the built bundle image and will be placed into the index database when the bundle is added to an index. 
+
+When OLM attempts to install the bundle, it will see the PDB and create it on-cluster. Since PDBs are namespace-scoped resources, 
+it will be created in the same namespace as the `InstallPlan` associated with the operator. The PDB will be visible in the `InstallPlan`
+and if the PDB fails to be installed OLM will provide a descriptive error in the `InstallPlan`. 
+
+When the operator is removed, the PDB will be removed as well via the kubernetes garbage collector. The PDB will be replaced when installing a newer version of the operator - 
+the newer PDB will replace the existing PDB on-cluster. 
+
+Prior versions of OLM (pre-0.16.0) will not support creating PDBs from the bundle. If a PDB is present in the bundle, OLM will throw an invalid installplan error
```

Comment:
If OLM detects a PDB should it check the MinKubeVersion and make sure it is greater than 0.16.0? Perhaps not since it would be difficult to maintain a map as new GVKs emerge. Thoughts?

Reply:
I'm not sure what you mean- this is referring to the OLM version not the kube version. I'm basically trying to state that older versions of OLM won't have this feature without saying OpenShift 4.6+ (to accommodate both upstream and downstream audiences).

Reply:
I understand the intent, but it is surprising to me that we're specifying the behavior of older versions in a design proposal for _newer_ versions. It makes me feel like we're changing something in older versions to get this behavior.

Might I suggest changing the wording to: "Prior versions of OLM **do not** support PDBs"